orion/sites/admin/src/programs/program.js

```diff
@@ -21,35 +21,93 @@ export class ProgramPage extends React.Component {
         });
     }
 
+    onClickSelectAll() {
+        var items = document.getElementsByName("unpublished");
+        for (var i = 0; i < items.length; i++) {
+            if (items[i].type == "checkbox") {
+                items[i].checked = true;
+            }
+        }
+    }
+
+    onClickPublish() {
+        console.log("clicked publish");
+    }
+
     render() {
         const rows = this.state.list.map((row, index) => {
             return <ProgramRow key={index} row={row} />;
         });
         const numRows = rows.length;
+        let numUnpublished = 0;
+        let numSelected = 0;
         return (
             <div id="view-program">
-                <h1>All Programs ({numRows})</h1>
+                <ul>
+                    <h1>All Programs ({numRows}) </h1>
+                    <p>
+                        {" "}
```

Comment:
what are these spaces for?

Reply:
I'm not sure, they just appeared after I ran prettify (I'll go ahead and delete them)